flexget/plugins/filter/list_match.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import PluginError
+
+log = logging.getLogger('list_match')
+
+
+class ListMatch(object):
+    schema = {
+        'type': 'object',
+        'properties': {
```

Comment:
indentation is quite ugly here, try to format it so that there's less indentation needed